main.py

```diff
@@ -1,10 +1,51 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as data_in:
+        players = json.load(data_in)
+    for name, player_info in players.items():
+        player_race_info = player_info["race"]
+        if not Race.objects.filter(name=player_race_info["name"]).exists():
+            player_race = Race.objects.create(
+                name=player_race_info["name"],
+                description=player_race_info["description"])
```

Comment:
```suggestion
                description=player_race_info["description"]
            )
```

Reply:
Small nit, but it will look more readable. Everything else is good)

Reply:
Yes, of course. Thank you